src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -6066,23 +6066,15 @@ private URL getResource( final List<String> classPath, final String resource )
                 getLog().error( "MalformedURLException: " + e.getMessage() );
             }
         }
-
-        URLClassLoader javadocClassLoader = new URLClassLoader( urls.toArray( new URL[urls.size()] ), null );
-        try
+        URL resourceURL = null;

```

Comment:
pull this and the return into the try block

Reply:
@elharo done.